projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -124,6 +124,13 @@ datajobs.status.watch.initial.delay=10000
 # On top of it the job source and its dependencies are installed for each job
 datajobs.deployment.dataJobBaseImage=python:3.9-slim
 
+# The map of python version and respective data job base and vdk images that would be
+# used for data job deployments
+datajobs.deployment.supportedPythonVersions=${DATAJOBS_DEPLOYMENT_SUPPORTED_PYTHON_VERSIONS:{}}
+# The default python version, which is to be used when selecting the vdk and job base images from
+# the supportedPythonVersions for data job deployments
+datajobs.deployment.defaultPythonVersion=${DATAJOBS_DEPLOYMENT_DEFAULT_PYTHON_VERSION:3.7}
```

Comment:
This default configuration is not correct. DefaultPythonVersion is 3.7 but you don't have any image for it. I think the best default version here is the one you used in dev profile.

Reply:
Changed, thanks